compliance/unapproved_instance_types/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.4
+
+- remove permissions block and adding incident resource table
```

Comment:
There were no changes to permissions block.

Reply:
Change the comment to reflect remove approval block and adding incident resource table